src/components/LinksForCentury.tsx

```diff
@@ -0,0 +1,65 @@
+import { Link, useLocation, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+
+const startedCenturies = ['16', '17', '18', '19', '20'];
+
+export const LinksForCentury: React.FC = () => {
+  const [searchParams] = useSearchParams();
+  const location = useLocation();
+
+  const centuries = searchParams.getAll('centuries') || [];
```

Comment:
Consider using enums or constants for string values to avoid errors connected with typos